doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-travel-agency-create-ocp-project.adoc

```diff
@@ -0,0 +1,48 @@
+[id='proc_kogito-travel-agency-create-ocp-project']
+
+= Creating an OpenShift project and installing the {PRODUCT} Operator using the {PRODUCT} CLI
+
+To set up an example {PRODUCT} application for deployment on {OPENSHIFT}, you must create a project (namespace) in OpenShift in which you can install the application and the {PRODUCT} Operator. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. The first time that you use the {PRODUCT} CLI to interact with a project or service, the {PRODUCT} Operator is automatically installed and used to execute the relevant tasks.
```

Comment:
I forgot to mention that to install operators in the OCP cluster, one needs to have `cluster-admin` rights.

Reply:
No, you did mention it, and it's already a prereq at the beginning of this tutorial, and in the other deploying sections before the tutorial.